ogame_test.go

```diff
@@ -2865,3 +2884,133 @@ func TestExtractOGameSession(t *testing.T) {
+	assert.Equal(t, "sr-en-152-f8286f9a8d053ad7df343d0873a98429e656f9e9", msgs[1].APIKey) //string
+	assert.Equal(t, "sr-en-152-55b7de6f4cd4772fd6c881fb3ea689f2f8d697d9", msgs[2].APIKey) //string
+	assert.Equal(t, "sr-en-152-08f7014551e9717de500fa2d52f4e19566a44299", msgs[3].APIKey) //string
+	assert.Equal(t, "sr-en-152-2fa03c5e6d96ffaf745d92abc0a44b5ad2d2c29b", msgs[4].APIKey) //string
+	assert.Equal(t, "sr-en-152-e7eb3f47e7e64df22ec8f96bd3232645321a727d", msgs[5].APIKey) //string
+
+	assert.Equal(t, float64(1),    msgs[0].LootPercentage) //float64
+	assert.Equal(t, float64(1),    msgs[1].LootPercentage) //float64
+	assert.Equal(t, float64(0.75), msgs[2].LootPercentage) //float64
+
+	assert.Equal(t, "2020-03-26 03:56:40", msgs[0].CreatedAt.Format("2006-01-02 15:04:05")) //time.Time / string
+	assert.Equal(t, "2020-03-26 03:54:29", msgs[5].CreatedAt.Format("2006-01-02 15:04:05")) //time.Time / string
+
+	assert.Equal(t, Coordinate{Galaxy:4, System:49, Position: 9, Type:3}, msgs[0].Target) //Coordinate
+	assert.Equal(t, Coordinate{Galaxy:4, System:49, Position: 9, Type:1}, msgs[1].Target) //Coordinate
+
+	assert.Equal(t, int64(0), msgs[0].Metal) //int64
+	assert.Equal(t, int64(0), msgs[0].Crystal) //int64
+	assert.Equal(t, int64(0), msgs[0].Deuterium) //int64
+
+	assert.Equal(t, int64(20159000), msgs[1].Metal) //int64
+	assert.Equal(t, int64(4363000), msgs[1].Crystal) //int64
+	assert.Equal(t, int64(4127000), msgs[1].Deuterium) //int64
+
+	assert.True(t, msgs[0].IsBandit) //bool
+	assert.False(t, msgs[0].IsStarlord) //bool
+	assert.True(t, msgs[1].IsBandit) //bool
+	assert.False(t, msgs[1].IsStarlord) //bool
+	assert.False(t, msgs[2].IsBandit) //bool
+	assert.False(t, msgs[2].IsStarlord) //bool
+	assert.False(t, msgs[3].IsBandit) //bool
+	assert.True(t, msgs[3].IsStarlord) //bool
+	assert.False(t, msgs[4].IsBandit) //bool
+	assert.True(t, msgs[5].IsStarlord) //bool
+	assert.False(t, msgs[5].IsBandit) //bool
+	assert.True(t, msgs[5].IsStarlord) //bool
+
+	assert.Equal(t, int64(0), msgs[0].DefenseValue)
+	assert.Equal(t, int64(0), msgs[0].FleetValue)
+	assert.True(t, msgs[0].HasDefenses) //bool
+	assert.Equal(t, int64(230692000), msgs[1].DefenseValue)
+	assert.Equal(t, int64(21985000), msgs[1].FleetValue)
+	assert.True(t, msgs[0].HasDefenses) //bool
+	assert.Equal(t, int64(46000), msgs[4].DefenseValue)
+	assert.Equal(t, int64(0), msgs[4].FleetValue)
+	assert.True(t, msgs[0].HasDefenses) //bool
+	assert.Equal(t, int64(1148000), msgs[5].DefenseValue)
+	assert.Equal(t, int64(5000000), msgs[5].FleetValue)
+
+
+	assert.True(t, msgs[0].HasDefenses) //bool
+	assert.True(t, msgs[0].HasFleet) //bool
+
+	assert.Equal(t, int64(0), msgs[0].CounterEspionage) //int64
+	assert.Equal(t, int64(100), msgs[1].CounterEspionage) //int64
+	assert.Equal(t, int64(98), msgs[5].CounterEspionage) //int64
+
+	assert.Equal(t, int64(60), msgs[0].LastActivity) //int64
+	assert.Equal(t, int64(60), msgs[1].LastActivity) //int64
+
+	assert.False(t, msgs[0].IsInactive) //bool
```

Comment:
Can you remove all the `// bool` comments. They add no value.
`assert.False` already compare boolean.
Same for all the other comments `// int64` / `// float64` ...

Reply:
I've removed them